packages/core/src/actions/order/cancel.ts

```diff
@@ -17,18 +18,19 @@ import { CancelInfo, FullCancelInfo } from "../../types";
 
 const cancelOrder = async (
   arkProvider: RpcProvider,
-  starknetAccount: Account,
+  starknetAccount: AccountInterface,
   arkAccount: Account,
-  cancelInfo: CancelInfo
+  cancelInfo: CancelInfo,
+  owner?: string
 ) => {
   const fullCancelInfo: FullCancelInfo = {
-    order_hash: cancelInfo.order_hash,
+    order_hash: cancelInfo.orderHash,
     canceller: starknetAccount.address,
     token_chain_id: shortString.encodeShortString("SN_MAIN"),
```

Comment:
We might allow other chain id, could this be set via options (defaulting to "SN_MAIN")